website/docs/r/container_cluster.html.markdown

```diff
@@ -75,9 +75,11 @@ output "cluster_ca_certificate" {
     in `initial_node_count` should be created in. Only one of `zone` and `region`
     may be set. If neither zone nor region are set, the provider zone is used.
 
-* `region` (Optional, [Beta](/docs/providers/google/index.html#beta-features))
+* `region` (Optional)
     The region to create the cluster in, for
     [Regional Clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-zone-and-regional-clusters#regional).
+    In a Regional Cluster, the number of nodes specified in `initial_node_count` is 
+    created in all zones of the region.
```

Comment:
Is this actually true? https://cloud.google.com/kubernetes-engine/docs/concepts/multi-zone-and-regional-clusters#overview says it's three zones.